salt/modules/restconf.py

```diff
@@ -0,0 +1,95 @@
+"""
+Execution module for Restconf Proxy minions
+
+:codeauthor: Jamie (Bear) Murphy <jamiemurphyit@gmail.com>
+:maturity:   new
+:platform:   any
+
+"""
+
+
+import logging
+
+__proxyenabled__ = ["restconf"]
+__virtualname__ = "restconf"
+
+log = logging.getLogger(__file__)
+
+
+def __virtual__():
+    if __opts__.get("proxy", {}).get("proxytype") != __virtualname__:
+        return False, "Proxytype does not match: {}".format(__virtualname__)
+    return True
+
+
+def info():
+    """
+        Should return some quick state info of the restconf device?
+    """
+    return "Hello i am a restconf module"
```

Comment:
Good point. I'm just not sure WHAT to put in here. other than may. Is this def a required def for proxy modules? I saw most seem to have it...

Reply:
At a glance it doesn't look like it's /required/. It might be useful to do something like https://tools.ietf.org/html/rfc8040#page-117 - and get the server capabilities :thinking:

Reply:
yea thats what I was thinking. will do

Reply:
done in https://github.com/ITJamie/salt/commit/46961404446033073c34dcd9ab2eea5945811246